### DISCUSSION TITLE
<Project Rock Paper and Scissors issue after commit to GitHub>

### DISCUSSION THREAD

**Discussion by Ha55an365** [community/community#131127] [Programming Help]
### Select Topic Area

Question

### Body

Hi all,

First time posting and have pushed my project for 'Rock, Paper, Scissors' from TheOdinProject to Github but it seems when I open the page, the pop-up where you type in either 'Rock', 'Paper' or 'Scissors is not working so I feel as if the Javascript is not working for some reason. Has anybody else had this issue?

**Comment by ebndev** [comment]
Hi @Ha55an365, thanks for being a part of the GitHub Community!

You are more likely to get a useful response if you share code snippets or a link to your repo. Giving a few more details might help someone give you a nudge in the right direction. 😄

**Comment by chiarorosa** [comment]
Gave us more information like..

Open your browser’s developer tools (usually F12 or right-click and select ‘Inspect’). Go to the ‘Console’ tab and see if there are any error messages. These messages can provide clues about what’s going wrong.

**Comment by CodewithWitcher** [comment]
share your repo it will help to get the solution.
